Upstream uses unfortunate variable name 'sun' which is defined as 1.

--- a/src/intf.c.orig	2023-12-11 19:03:19.685141153 +0100
+++ b/src/intf.c	2023-12-11 19:03:29.441538809 +0100
@@ -668,6 +668,7 @@
 int
 intf_get_dst(intf_t *intf, struct intf_entry *entry, struct addr *dst)
 {
+#undef sun
 	union sockunion sun;
 	socklen_t n;
 
